app/javascript/components/OrderManager.vue

```diff
@@ -27,7 +27,7 @@
       Sorry, there was a problem saving your work!
     </alert>
     <wrapper
-      full-width="true"
+      :full-width="true"
```

Comment:
This was introduced in order to ensure that the "true" property value supplied here is cast to a Boolean.